src/components/molecules/CategoryEdit/index.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div class="category-edit">
+    <app-heading
+      class="category-edit__head"
+    >
+      カテゴリー管理
+    </app-heading>
+    <app-router-link
+      class="category-edit__return"
+      underline
+      hover-opacity
+      :to="`/categories`"
+    >
+      カテゴリー一覧へ戻る
+    </app-router-link>
+    <app-input
+      v-validate="'required'"
+      class="category-edit__input"
+      name="category"
+      type="text"
+      :value="editCategoryName"
+      @update-value="$emit('edit-value', $event)"
+    />
+    <app-button
+      class="category-edit__button"
+      button-type="submit"
+      round
+      @click="editClick(editId, editCategoryName)"
+    >
+      更新
+    </app-button>
+
+    <div v-if="doneMessage" class="category-edit__doneMessage">
+      <app-text bg-success>{{ doneMessage }}</app-text>
+    </div>
+    <div v-if="errorMessage" class="category-edit__errorMessage">
+      <app-text bg-error>{{ errorMessage }}</app-text>
+    </div>
+  </div>
+</template>
+
+<script>
+import {
+  RouterLink, Heading, Button, Input, Text,
+} from '@Components/atoms';
+
+export default {
+  components: {
+    appRouterLink: RouterLink,
+    appHeading: Heading,
+    appButton: Button,
+    appInput: Input,
+    appText: Text,
+  },
+  props: {
+    theads: {
+      type: Array,
+      default() {
+        return [];
+      },
+    },
+    editId: {
+      type: Number,
+      default: null,
+    },
+    editCategoryName: {
+      type: String,
+      default: '',
+    },
+    doneMessage: {
+      type: String,
+      default: '',
+    },
+    errorMessage: {
+      type: String,
+      default: '',
+    },
+    access: {
+      type: Object,
+      default: () => ({}),
+    },
+  },
+  methods: {
+    editClick(editId, editCategoryName) {
+      this.$emit('edit-click', editId, editCategoryName);
```

Comment:
[imo]

基本的に対象が同じものの情報についてはオブジェクトで管理したほうが可読性が上がります。
また、メソッド名からどんな操作をしたいかはわかると思うので、変数名はシンプルにできるかなと思いました

```suggestion
    editClick(id, name) {
      this.$emit('edit-click', { id, name });
```

Reply:
シンプルな名前に変更しました。